pytripgui/controller/plot_cont.py

```diff
@@ -0,0 +1,34 @@
+import logging
+import pytrip as pt
+
+from PyQt5.QtCore import QObject, pyqtSignal, pyqtSlot
+
+logger = logging.getLogger(__name__)
+
+
+class PlotController(object):
+    """
+    This class holds all logic for plotting the canvas.
+    """
+    def __init__(self, model, fig):
+        self.model = model
+        self.fig = fig
+
+    @pyqtSlot()
+    def update_plot(self):
+        """
+        Updating plot.
+        What has to be plotted could be defined in model/. ? And in a seperate file?
```

Comment:
The data to be plotted (i.e. ctx cube should go into the model), colors, look&feel ("how it is plotted" should be defined here).